src/objects.py

```diff
@@ -192,6 +192,35 @@ def net_cash_df(self) -> pd.DataFrame:
         
         return df
 
+class PlayerRating:
```

Comment:
yeah i don't really know classes that well, so went with what I know like `__init__` but there may be a better way to do it that you know:)